build.assets/tooling/cmd/get-webapps-version/main.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command get-webapps-version determines the appropriate version
+// of webapps to check out for a given version of teleport.
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func main() {
+	fmt.Println(webappsVersion(
```

Comment:
Should this strip whitespaces (or just fmt.Print instead) so the caller doesn't have to deal with newlines?